qt-downloader

```diff
@@ -335,8 +338,19 @@ def main():
     if info is None:
       sys.exit(1)
 
+    if args.mingw:
+      mingw = toolchain_build_url(args, 'mingw')
+      mingw_name, mingw_version, mingw_archives = mingw_info(mingw, args.mingw)
+      download_and_extract(mingw + mingw_name + '/' + mingw_version, mingw_archives, args.output, [])
+
+    if args.openssl:
+      ossl = toolchain_build_url(args, args.openssl)
+      ossl_name, ossl_version, ossl_archives = ossl_info(ossl, args.openssl)
+      download_and_extract(ossl + ossl_name + '/' + ossl_version, ossl_archives, args.output, [])
+
```

Comment:
The above two `if`s are likely to be a part of the _Allow to download Specific Toolchains_ commit.